wolfcrypt/src/asn.c

```diff
@@ -31643,7 +31643,7 @@ int wc_SetExtKeyUsageOID(Cert *cert, const char *in, word32 sz, byte idx,
         void* heap)
 {
     byte oid[MAX_OID_SZ];
```

Comment:
Change size of oid too.

Reply:
Thank you, I was silly and missed that (-: